src/diffChecker.test.ts

```diff
@@ -157,6 +196,117 @@ describe('diffChecker', () => {
     });
   });
 
+  describe('new file with no coverage above threshold', () => {
+    beforeEach(() => {
+      // Set coverageDecreaseThreshold to 100% so we only test coverageThreshold.
+      result = diffChecker(
+        fileFullCovered,
+        newFileNotCovered,
+        undefined,
+        40,
+        100,
+        40
+      );
+    });
+
+    it('should not regress', () => {
+      // Total is now 50% but the first file didn't change so no regression
+      expect(result.regression).toBe(false);
+    });
+
+    it('should be below threshold', () => {
+      // Total is now 50% but file threshold is 40%, and new file threshold is 40, so it should be below the threshold
+      expect(result.belowThreshold).toBe(true);
+    });
+
+    it('outputs correct total percentages', () => {
+      expectToMatchObject(result.totals, {
+        belowThreshold: false,
+        decreased: false, // Threshold is 40
+        pcts: {
+          branches: 50,
+          functions: 50,
+          lines: 50,
+          statements: 50
+        }
+      });
+    });
+  });
+
+  describe('new file with half coverage below threshold', () => {
+    beforeEach(() => {
+      // Set coverageDecreaseThreshold to 100% so we only test coverageThreshold.
+      result = diffChecker(
+        fileFullCovered,
+        newFileHalfCovered,
+        undefined,
+        40,
+        100,
+        60
+      );
+    });
+
+    it('should not regress', () => {
+      // Total is now 75% and the first file didn't change so no regression
+      expect(result.regression).toBe(false);
+    });
+
+    it.only('should be below threshold', () => {
```

Comment:
Leftover it.only. Also I looked around and Prettier will most likely fail, make sure you run a fix for that also.

I made some changes to the CI flow and I hope it triggers on the next push on this PR. I was also trying to implement a `/test` command but it doesn't get the reference of the branch so it's not good atm.

Reply:
woops... Yea We have an ESLint rule to throw an error on this. I should probably re-run the tests too...

Reply:
Removed in df2b7d03aef0f2f69d51a4869ef88ede9100e1e6